resources/inspectionDescriptions/LatexPackageMayNotExist.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<body>
+The package given in a <tt>\usepackage</tt> or <tt>\RequirePackage</tt> command may not exist.
```

Comment:
Perhaps it would be clearer to say something like 'Package was not found on CTAN or locally'? This description sounds a bit general, like something you can say in general about any LaTeX document: of course a package may not exist, the point is that for this package you included right here we cannot find it.

Reply:
Perhaps 'package not found' is more consistent than 'package may not exist'

Reply:
Yeah, that's probably better.